api/src/main/java/org/openmrs/module/pihcore/config/registration/ContactPersonConfigDescriptor.java

```diff
@@ -0,0 +1,30 @@
+package org.openmrs.module.pihcore.config.registration;
+
+import org.codehaus.jackson.annotate.JsonProperty;
+
+public class ContactPersonConfigDescriptor {
+
+	@JsonProperty
+	private Boolean required;
+
+	@JsonProperty
+	private RegistrationFieldDescriptor phoneNumber;
+
+	public Boolean getRequired() {
```

Comment:
I decided to extract the "required" configuration out to the PIH Config as well while I was at it.